Update gh-bb-integration.md (#4287)
circleci/circleci-docs: jekyll/_cci2/gh-bb-integration.md

It is possible to edit the webhooks in GitHub or Bitbucket Cloud to restrict events that trigger a build. Editing the webhook settings lets you change which hooks get sent to CircleCI, but doesn't change the types of hooks that trigger builds. CircleCI will always build push hooks and will build on PR hooks (depending on settings), but if you remove push hooks from the webhook settings CircleCI won't build. Refer to the [GitHub Edit a Hook document](https://developer.github.com/v3/repos/hooks/#edit-a-hook) or the [Atlassian Manage Webhooks document](https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html) for details.

> **Note** Currently, CircleCI only supports GitHub and Bitbucket for git. If you wish to use AWS CodeDeploy in your git integrations, you may use the [AWS Code Deploy](https://circleci.com/orbs/registry/orb/circleci/aws-code-deploy) orb in the Orbs Registry. You may also add a manual `git clone` into your job if so desired.

Reviewer: ```suggestion
**Note** Currently, CircleCI supports GitHub and Bitbucket version control systems. If you wish to use AWS CodeDeploy in your git integrations, you may use the [AWS Code Deploy](https://circleci.com/orbs/registry/orb/circleci/aws-code-deploy) orb in the Orbs Registry. You may also add a manual `git clone` into your job if so desired.
```

Just a suggestion to use VCS instead of `git` -- also, the original [issue](https://github.com/circleci/circleci-docs/issues/4224) is referring to CodeCommit -- but this is referencing AWS CodeDeploy? I'm not sure if that's what you mean?

Reviewer 2: Agreed, CodeCommit is the AWS git hosting. We don't have a CodeCommit orb as far as I can tell.
